src/functions/apiFunctions.tsx

```diff
@@ -1,5 +1,42 @@
-import { ArtistData } from "@/interfaces/artistInterface";
+import { GenreData } from "@/interfaces/genreInterface";
 import { ResponseData } from "@/interfaces/songInterface";
+import { ArtistData } from "@/interfaces/artistInterface";
+
+export function tallyArtist(data: ResponseData): Map<string, number> {
+  let artistCounts = new Map<string, number>();
+
+  if (data.items) {
+    data.items.forEach((item) => {
+      item.artists.forEach((artist) => {
+        if (artistCounts.has(artist.name)) {
+          artistCounts.set(artist.name, artistCounts.get(artist.name)! + 1);
+        } else {
+          artistCounts.set(artist.name, 1);
+        }
+      });
+    });
+    return artistCounts;
```

Comment:
I don't think this return does anything as it lives in a `forEach`

Reply:
this one was outside the foreach but inside the if statement? deleted anyways because confirmed data is the same